update readme (#6)
natakazakova/asp-net-core-dashboard-custom-item-gallery: readme.md

![online-map-item](images/online-map-item.png)

- **Provider** - Specifies whether to show Azure or Bing maps.
- **Type** - Specifies the map type. You can choose between `RoadMap`, `Satellite` or `Hybrid`.
- **Display Mode** - Specifies whether to show markers or routes.

> Please specify the credentials for Azure/Bing Maps to ensure maps render correctly. 

Reviewer: Maybe highlight [the line where we must add created apikey](https://github.com/DevExpress-Examples/asp-net-core-dashboard-custom-item-gallery/blob/6cc76d6bae9eeb998887f2b81b6a1baec340aba2/CS/AspNetCoreCustomItem/wwwroot/js/OnlineMapItem.js#L196)?

Author: ```suggestion
Please specify the credentials for Azure/Bing Maps to ensure maps render correctly. You can specify the credentials in the [OnlineMapItem.js](CS/AspNetCoreCustomItem/wwwroot/js/OnlineMapItem.js#L194) file.
```

> Please specify the credentials for Azure/Bing Maps to ensure maps render correctly. You can specify the credentials in the [OnlineMapItem.js](CS/AspNetCoreCustomItem/wwwroot/js/OnlineMapItem.js#L194) file.
> 

Author: ```suggestion
To ensure maps render correctly, specify your Azure/Bing Maps credentials in the [OnlineMapItem.js](CS/AspNetCoreCustomItem/wwwroot/js/OnlineMapItem.js#L194) file.

```
